components/Button/StyledButton.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Button from 'antd/lib/button';
+import styled from 'styled-components';
+import 'antd/lib/button/style';
+
+const OverrideStyle = styled.div`
```

Comment:
Probably just request `color, backGroundColor` and so on directly, and define a default property in case styling isn't passed since otherwise this would break the component.